lib/AppInfo/Application.php

```diff
@@ -80,7 +81,7 @@ public function __construct(array $urlParams = []) {
 				$c->query('Environment'),
 				$c->query('OCP\IURLGenerator'),
 				$c->query('OCP\IConfig'),
-				$c->query('Session')
+				$c->query(EventDispatcherInterface::class)
```

Comment:
Because it wasn't part of the signature php just ignored the extra paramenter: https://github.com/nextcloud/gallery/blob/master/lib/Controller/PageController.php#L53-L58